README.md

```diff
@@ -585,3 +585,60 @@ you need to define a `key` or other `tag` with a "star" query syntax like
 `values.*`. This *hack* is in place because it's not allowed to use `*` in the
 metric label definitions. If both annotations and corresponding label is
 defined, then the annotation takes precedence.
+
+## HTTP Collector
+
+The http collector allows collecting metrics from an external endpoint specified in the HPA.
+Currently only `json-path` collection is supported.
+
+### Supported metrics
+
+| Metric | Description | Type | K8s Versions |
+| ------------ | -------------- | ------- | -- |
+| *custom* | No predefined metrics. Metrics are generated from user defined queries. | Pods | `>=1.12` |
+
+### Example
+
+This is an example of using the HTTP collector to collect metrics from a json
+metrics endpoint specified in the annotations.
+
+```yaml
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: myapp-hpa
+  annotations:
+    # metric-config.<metricType>.<metricName>.<collectorName>/<configKey>
+    metric-config.external.http.json/json-key: "$.some-metric.value"
+    metric-config.external.http.json/endpoint: "http://metric-source.app-namespace:8080/metrics"
+    metric-config.external.http.json/aggregator: "max"
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: myapp
+  minReplicas: 1
+  maxReplicas: 10
+  metrics:
+  - type: External
+    external:
+      metric:
+        name: http
+        selector:
+          matchLabels:
+            identifier: unique-metric-name
+      target:
+        averageValue: 1
+        type: AverageValue
+```
+
+The HTTP collector similar to the Pod Metrics collector. The metric name should always be `http-json`.
```

Comment:
this does not match the yaml example above, it it intended?

Reply:
Fixed